README.md

```diff
@@ -84,11 +84,11 @@ Now you can launch the container by issuing:
 
     docker-compose up -d
 
-### Enviroment values
-**ENV WWW_PORT=8080** - Specify default HTTP port
-**ENV SSL_PORT=443** - Specify default SSL port
-**ENV TZ=Europe/Amsterdam** - Specify default timezone (see /usr/share/zoneinfo folder), **only needed when you can not mount the volume /etc/localtime**
-**EXTRA_CMD_ARG** - Option to override additional command line parameters (See domoticz --help)
+### Environment values
+**ENV WWW_PORT=8080** - Specify default HTTP port  
```

Comment:
Is it possible to remove the extra (white) spaces at the end of this line, and the lines below as well ?

Reply:
The extra white spaces at the end are triggering a new line (github md file behavior). So now you get a nice list of values.

Reply:
Thanks! It just looked strange on Github ;)